slice/IceGrid/Exception.ice

```diff
@@ -17,21 +17,21 @@
 
 module IceGrid
 {
-    /// This exception is raised if an application does not exist.
+    /// The exception that is thrown when IceGrid does not know an application with the provided name.
     exception ApplicationNotExistException
     {
         /// The name of the application.
         string name;
     }
 
-    /// This exception is raised if a server does not exist.
+    /// The exception that is thrown when IceGrid does not know a server with the provided server ID.
     exception ServerNotExistException
     {
         /// The identifier of the server.
```

Comment:
We need to review how we document "id" fields. In this file we use:

* The identifier
* The id
* The identity

Reply:
Yes. I think it should be ID (adapter ID, server ID) or identity. Never identifier.

Reply:
Fixed